testing/run-test-locally.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "$DIR" || exit 3
+
+docker network create --driver bridge phpmyadmin
+
+function restore() {
+    docker kill phpmyadmin
+    docker kill mariadb
+    exit 99
+}
+
+trap restore 0 1 2 3 6 14 15
+
+docker kill mariadb
+docker kill phpmyadmin
+
+set -e
+docker run --rm -d -p 3300:3306 -e MYSQL_ROOT_PASSWORD=my-secret-pw --init --hostname mariadb --name mariadb --network phpmyadmin --tmpfs /var/lib/mysql:rw,noexec,nosuid,size=300m mariadb:10.3
```

Comment:
coud you use some ENV variables that have default values like `${DOCKER_MYSQL_PORT:-3306}` ?